### PR TITLE
Added new gamemode, time left alert

### PR DESCRIPTION
**Pull request by ItzBlack6093**
- Added time left alert
image

- New Gamemode: "Climb"
_This is TETR.IO Quick Play 2 but for teti, for hyperspeed practice i guess..._

### CODE REVIEW

File: src/display/rendering.js [hunk 1]
```diff
@@ -314,6 +327,8 @@ export class Rendering {
         this.game.boardEffects.move(0, 0);
         this.game.boardEffects.rotate(0);
         this.game.boardEffects.rainbowBoard();
+        if(this.game.settings.game.gamemode == "ultra" && this.game.stats.time == 60.00000000000378) this.renderTimeLeft("60S LEFT")
```
Comment (ItzBlack6093) [inline src/display/rendering.js:330]: the game's time system is a bit stupid xd

Comment (TitanPlayz100) [reply src/display/rendering.js:330]: eh its probably fine, could use rounding instead

**Comment by TitanPlayz100** [comment]
great implementation, especially love how you recreated the flashing text and the climb bar.

remember to pull main so that stuff is synced

**Comment by ItzBlack6093** [comment]
about the grade system, i'm not doing it right now cus i'm busy for studying
maybe next update...

**Comment by ItzBlack6093** [comment]
image
_F10 Any% WR_

**Comment by TitanPlayz100** [comment]
ok wow that looks good (feel free to change the font to something more readable for timers)

**Comment by TitanPlayz100** [comment]
Wait I forgor about teti💀

**Comment by ItzBlack6093** [comment]
bro this is your game 💀 

_not pulling right now cuz positioning the container is very hard_

**Pull request merged**